src/graphql/queries/Profiles.graphql

```diff
@@ -0,0 +1,7 @@
+query Profiles($request: ProfileQueryRequest!) {
+  profiles(request: $request) {
+    items {
+      ...ProfileFields
```

Comment:
Are we sure we need all these fields? Would be nice to pare down to just what is necessary for the preview.

Reply:
Yeah, I think `bio` is the only thing we aren't using currently. We run the risk of the fragment bloating in the future, but gain confidence we'll have enough data to populate any components that depend on Profile. I thought it was an ok tradeoff since it's slim, but will defer to @bigint on his preference 👀 .

```graphql
fragment ProfileFields on Profile {
  id
  name
  handle
  bio
  ownedBy
  isDefault
  attributes {
    key
    value
  }
  picture {
    ... on MediaSet {
      original {
        url
      }
    }
    ... on NftImage {
      uri
    }
  }
  followModule {
    __typename
  }
}
```